core/src/main/java/org/mule/expression/DefaultExpressionManager.java

```diff
@@ -604,6 +604,14 @@ public void initialise() throws InitialisationException
     {
         MVELExpressionLanguage mel = new MVELExpressionLanguage(muleContext);
         mel.initialise();
+        try
```

Comment:
This is is hack. Start should not need to be invoked as part of a initialise phase. First you split initialise/start in MVELExpressionLanguage because you needed it in that way, now you are invoking both methods here.